packages/core-magistrate-crypto/src/transactions/bridgechain-update.ts

```diff
@@ -54,7 +56,7 @@ export class BridgechainUpdateTransaction extends Transactions.Transaction {
         }
 
         const buffer: ByteBuffer = new ByteBuffer(4 + 1 + seedNodesBuffersLength + seedNodes.length, true);
```

Comment:
This should be `32` instead of `4` now, right? Afaik the four bytes were reserved for the bridgechain id, which was an unsigned 32bit integer.

Reply:
Yep. See https://github.com/ArkEcosystem/core/pull/3284